hardhat/dist/hardhat.config.js

```diff
@@ -70,23 +70,13 @@ const tokens = {
         });
```

Comment:
this should have been included in the commit that changed mintWeth in the TS code